### ISSUE TITLE
Error - Notice: Undefined property: Phalcon\Config::$layoutsDir

### ISSUE THREAD

**Issue by alejosv** [mrcoco/vokuro-cms#1]
https://github.com/mrcoco/vokuro-cms/blob/a4821a3e69bcc65f2fd8c2be64b0798c1031448e/Modules.php#L47

**Comment by mrcoco** [comment]
do you use vokuro-modular( https://github.com/mrcoco/vokuro-modular.git). vokuro-cms , vokuro-cms is part of vokuro-modular ?

**Comment by alejosv** [comment]
Yes, the vokuro-cms is installed when it is run `composer install` into vokuro-modular
